CHANGELOG-PENDING.md

```diff
@@ -5,6 +5,7 @@
 ### BREAKING CHANGES
 
 ### FEATURES
+- [adr: add adr for bad encoding fraud proof](https://github.com/celestiaorg/celestia-node/pull/492) [@vgonkivs](https://github.com/vgonkivs)
```

Comment:
The pending changelog is being removed as of #500, so maybe it would be a good idea not to include this line to save on merge conflicts.